halotools/mock_observables/pair_counters/npairs_jackknife_3d.py

```diff
@@ -215,19 +219,14 @@ def _npairs_jackknife_3d_process_weights_jtags(sample1, sample2,
             raise HalotoolsError("weights2 should have same len as sample2")
 
     # Process jtags_1 entry and check for consistency.
-    if jtags1 is None:
-        jtags1 = np.array([0]*np.shape(sample1)[0], dtype=np.int)
-    else:
-        jtags1 = np.asarray(jtags1).astype("int")
-        if np.shape(jtags1)[0] != np.shape(sample1)[0]:
-            raise HalotoolsError("jtags1 should have same len as sample1")
+    jtags1 = np.asarray(jtags1).astype("int")
+    if np.shape(jtags1)[0] != np.shape(sample1)[0]:
+        raise HalotoolsError("jtags1 should have same len as sample1")
+
     # Process jtags_2 entry and check for consistency.
-    if jtags2 is None:
```

Comment:
If this is true, we set them to `0`s but then later error if they are `< 1` and so immediately error. Essentially this means that they are required which is what I have changed the docs + code to reflect.

Reply:
Yup, this is what was intended.